crispy_forms/bootstrap.py

```diff
@@ -386,3 +388,9 @@ def __init__(self, field, *args, **kwargs):
 
 class InlineField(Field):
     template = "%s/layout/inline_field.html"
+
+if django.VERSION < (1, 9):
```

Comment:
Hmmm. This wasn't quite what I thought you wanted... 

If we do it this was `ReadOnlyField` won't be available at all in 1.9 — the `disabled` attribute will do all. 

Is that correct? Is it enough? I'm happy to do it this way if so. 

(I had thought in `__init__` you'd set `readonly` for 1.8 and `disabled` for 1.9+)